### ISSUE TITLE
Update for iOS 12.4? Message stuck at Sending... screen

### ISSUE THREAD

**Issue by fanium** [runnersaw/pebble-text#20]
Hi Sawyer, 
Jailbreak for iOS 12.4 was released yesterday. I found out messages can not send out from Pebble Steel and stuck at the "Sending..." screen. Your app works perfectly under iOS11 also iOS10. 
Could you help to update it for iOS 12.4?

Thank you in advance.

**Comment by CobreDev** [comment]
@joemerlino is working on trying to update it (#19). We can't guarantee that he will be successful, but he's taking a look

**Comment by fanium** [comment]
Good news! The current version works perfectly after I updated the unc0ver from 3.5.5 to 3.6.3
I learned this from Reddit, and I tested both PS and PTS, both pebble's send text and TextSender, all works. Yeah~~~

Based on the unc0ver release log [https://github.com/pwn20wndstuff/Undecimus/releases](https://github.com/pwn20wndstuff/Undecimus/releases) , there is a major change at 3.6.0, but I updated directly to 3.6.3. Not sure 3.6.0~ 3.6.2 works or not, but who cares, just update to the 3.6.3.

so issue is closed, thank you very much every one. cheers

**Comment by joemerlino** [comment]
Awesome news! sorry that I was not able to deliver it earlier

**Comment by fanium** [comment]
> Awesome news! sorry that I was not able to deliver it earlier

Hi joemerlin, 
Are you able to recompile the project? I am thinking maybe it can support the reply function for more IM applications such as Wechat, and so on.  This is an awesome application makes the pebble watch better.
thanks,

**Comment by CobreDev** [comment]
@fanium What version of TextSender are you using?

**Comment by fanium** [comment]
I am using 3.0.0

**Comment by cdlenfert** [comment]
Hi @joemerlino. Same question about possibly picking this up again and expanding the functionality to include other message replies outside of iMessages. I've also found that unc0ver which fixes iMessage functionality in TextSender unfortunately crashes my phone every day or two (that is without anything but the jailbreak installed .. and is a known issue by Pwn20wnd). I switched to the checkra1n jailbreak and it's much more stable (no freezes or crashes). The downside is that TextSender doesn't work at all on checkra1n (0.9.6 beta currently).
